pkg/api/plugins.go

```diff
@@ -368,21 +370,25 @@ func (hs *HTTPServer) InstallPlugin(c *models.ReqContext) response.Response {
 	}
 	pluginID := web.Params(c.Req)[":pluginId"]
 
-	err := hs.pluginManager.Add(c.Req.Context(), pluginID, dto.Version)
+	err := hs.pluginManager.Add(c.Req.Context(), pluginID, dto.Version, plugins.CompatOpts{
+		GrafanaVersion: hs.Cfg.BuildVersion,
```

Comment:
Isn't this always the same value, both here and in cli (services.GrafanaVersion)? I.e. the version injected when building Grafana? I guess this related to your description (below), but I don't understand what you mean, sorry.

> Public grafana plugin "SDK" now accepts "compatibility options" when installing plugins. Previously, we were hardcoding this info and using Grafana server specific config, which hurts portability (now we're closer to not depending on any specific Grafana config within the plugins package)

Reply:
The intention here is to make the APIs more flexible depending on the client. Core grafana will always have the same version, but in other cases, maybe/maybe not 😉

Reply:
Got it :+1: